### DISCUSSION TITLE
New mods ideas

### DISCUSSION THREAD

**Discussion by uuigdevt47** [ppy/osu#16723] [Ideas] [answered]
I'd like to see start pos mod to start from some part of level to train it, and also it will be good to see cursor dance autoplay bot!

**Comment by mk56-spn** [comment, marked as answer]
this has already been suggested to death #15099 #13013 #13076, ppy has already shared his interest in having some sort of practice mode, so it will likely come

**Reply by mk56-spn** [reply]
as for the cursor bot dance autoplay this seems sufficiently covered by other clients like Danser which have specialized it more than lazer could without making auto into a mess of toggles so  idk i dont see that happening
